account_operating_unit/models/account_move.py

```diff
@@ -12,6 +12,7 @@ class AccountMoveLine(models.Model):
     operating_unit_id = fields.Many2one(
         comodel_name="operating.unit",
     )
+    is_ou_balance = fields.Boolean()
```

Comment:
```suggestion
    is_ou_balance = fields.Boolean(readonly=True)
```

Just to be safe :) The rest LGTM, I do not see other way to identify them without the flag.

Would be nice to have a migration script to fill old values

Reply:
Nice suggestions :) added both of them